exchanges/btcmarkets/btcmarkets_wrapper.go

```diff
@@ -546,7 +547,7 @@ func (b *BTCMarkets) GetActiveOrders(req *order.GetOrdersRequest) ([]order.Detai
 	}
 	var err error
 	for x := range req.Currencies {
-		tempData, err = b.GetOrders(b.FormatExchangeCurrency(req.Currencies[x], asset.Spot).String(), -1, -1, -1, "")
+		tempData, err = b.GetOrders(b.FormatExchangeCurrency(req.Currencies[x], asset.Spot).String(), -1, -1, -1, "open")
```

Comment:
const "open" and change string in get all orders

Reply:
consting provides no real benefit (folding, propagation etc, sure its immutable but you aren't changing the value of open at run time and since its a const and passed to a method its always passed by value anyway)

Since GetOrders can only ever be open or all (or blank that defaults to all) it might even be worth while having this as a bool for openOnly over a string then GetOrders can just be 
```go
	if openOnly {
		params.Set("status", "open")
	}
```

what are your thoughts on this? @shazbert @thrasher- @MadCozBadd

Reply:
I am happy with a bool, I also enjoy a good consting.

Reply:
I am happy either way both have benefits

Keeping it a string keeps it inline with what their API expects (and if they ever add other options in future its easier to add)

Having a bool is less error prone and less bytes passed across

Reply:
bool seems good imo

Reply:
pushed up a change to move this over to a bool
By default status is not sent all records are returned so take advantage of that